### PR TITLE
TS-4805: CID 1021743: Uninitialized scalar field

### PR DESCRIPTION
**Pull request by bryancall**

### CODE REVIEW

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/664/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/559/ for details.

**Comment by zwoop** [comment]
:+1:

File: iocore/net/UnixNet.cc [hunk 1]
```diff
@@ -289,7 +289,20 @@ initialize_thread_for_net(EThread *thread)
...
+    active_queue_size(0),
+    max_connections_per_thread_in(0),
+    max_connections_active_per_thread_in(0),
+    max_connections_in(0),
+    max_connections_active_in(0),
+    inactive_threashold_in(0),
+    transaction_no_activity_timeout_in(0),
+    keep_alive_no_activity_timeout_in(0),
+    sec(0),
+    cycles(0)
```
Comment (jpeach) [inline iocore/net/UnixNet.cc]: Remove ``sec`` and ``cycles``. They aren't used AFAICT.

**Pull request merged**